fury/tests/test_actors.py

```diff
@@ -537,6 +537,26 @@ def test_odf_slicer(interactive=False):
     del odf_actor
     del odfs
 
+def test_peak_symmetric(interactive=False):
```

Comment:
There is no need to test this in the `test_actors.py` since this is a functionality of the peak actor, just keep the test in the corresponding (`test_peak.py`) test file.